cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -304,6 +312,79 @@ impl EdgeAddRemovePlugin {
 
         Ok(())
     }
+
+    fn add_conditional_edges(&self, graph: &mut cincinnati::Graph) -> Fallible<()> {
+        let mut conditional_edges: Vec<ConditionalEdge> = vec![];
+        graph
+            .conditional_edges
+            .iter()
+            .try_for_each(|ce| -> Fallible<()> {
+                let mut conditional_edge = ConditionalEdge {
+                    edge_regex: ce.edge_regex.clone(),
+                    edges: vec![],
+                    risks: ce.risks.clone(),
+                };
+
+                let to_regex_string = &ce.edge_regex.to;
+                let from_regex_string = &ce.edge_regex.from;
+
+                let to_regex = regex::Regex::new(to_regex_string)
+                    .context(format!("Parsing {} as Regex", to_regex_string))?;
```

Comment:
At the moment, [`from` is a regexp, but `to` is a non-regexp release reference][1].  `to` will need [arch-suffix handling][2], and the regexp in `from` is expected to apply [before the arch suffix is stripped][3]

[1]: https://github.com/openshift/cincinnati-graph-data/tree/bfd85c895952e2517b5593798c3a7b2c51faff43#block-edges
[2]: https://github.com/openshift/cincinnati-graph-data/tree/bfd85c895952e2517b5593798c3a7b2c51faff43#release-names
[3]: https://github.com/openshift/cincinnati-graph-data/blame/bfd85c895952e2517b5593798c3a7b2c51faff43/README.md#L134-L137

Reply:
changed it to a string